README.md

```diff
@@ -4,79 +4,73 @@ Gqrx
 
-It is strongly recommended to run the `volk_profile` utility before
-running gqrx. This will detect and enable processor-specific optimisations and
-will in many cases give a significant performance boost.
+Installation 
+============
 
-The first time you start gqrx it will open a device configuration dialog.
-Supported devices that are connected to the computer are discovered
-automatically and you can select any of them in the drop-down list.
+Our installation instructions cover Linux, Mac OS, Raspberry-Pi, and source code. If you're installation option is not available here, you can request it via [this 1 question poll](https://poll.fm/11102781).
 
-If you don't see your device listed in the drop-down list it could be because:
-- The driver has not been included in a binary distribution
-- The udev rule has not been properly configured
-- Linux kernel driver is blocking access to the device
+Linux distributions
+-------------------
 
-You can test your device using device specific tools, such as rtl_test,
-airspy_rx, hackrf_transfer, qthid, etc.
+The simplest installation for Linux distributions is by our [official AppImage release](https://github.com/gqrx-sdr/gqrx/releases). Many Linux distributions provide gqrx in their package repositories. Linux distribution versions can be out-of-date. 
 
-Gqrx supports multiple configurations and sessions if you have several devices
-or if you want to use the same device under different configurations. You can
-load a configuration from the GUI or using the `-c` command line argument. See
-`gqrx --help` for a complete list of command line arguments.
+Alternatively you can install from [source code](https://github.com/gqrx-sdr/gqrx/edit/master/README.md#installation-from-source). It will soon be possible to install gqrx on [linux-64, linux-aarch64 via the conda-forge](https://github.com/gqrx-sdr/gqrx/edit/master/README.md#installation-via-conda).
 
-Tutorials and howtos are being written and published on the website
-https://gqrx.dk/
+Mac OS install 
+--------------
 
+The simplest installation is by our [offical Mac OS release](https://github.com/gqrx-sdr/gqrx/releases).
 
-Known problems
---------------
+Alternate Mac support is available through [MacPorts](https://ports.macports.org/port/gqrx/details/) and [Homebrew](https://formulae.brew.sh/cask/gqrx). It will soon be possible to install gqrx on [Mac OS osx-64 via the conda-forge](https://github.com/gqrx-sdr/gqrx/edit/master/README.md#installation-via-conda).
 
-See the bug tracker on Github: https://github.com/gqrx-sdr/gqrx/issues
+Raspberry-Pi
+------------
 
+The simplest installation for Raspberry-Pi devices is via the Raspberry-Pi software repository.
 
-Getting help and reporting bugs
--------------------------------
+```
+open the Applications menu
+select Preferences
+Add / Remove Software
+search for Gqrx
+After installation, Gqrx will be available in the Internet section of the Applications menu
```

Comment:
I would like to recommend to rephase this point, on ArchLinux(Arm) with [hamrado-menu](https://aur.archlinux.org/packages/hamradio-menus) installed, gqrx shows up in the hamradio section..
I dont know how it is on other distributions.

Reply:
Ok so either specify these instructions are for Ras-Pi OS, or make them generic, ala "then search for gqrx to start it."

I think the recommended way would be to specify each OS, but this will make the readme longer.

In which case I think it'd be better to break these instructions out into an INSTALL.md?